nemo/collections/nlp/models/language_modeling/megatron_gpt_prompt_learning_model.py

```diff
@@ -185,6 +188,32 @@ def init_model(self, cfg: DictConfig, trainer: Trainer):
         self.lowest_val_loss = None
         self.prompt_encoder = None
 
+        # default inference related params -> for evaluation metrics
+        self.length_params: LengthParam = {
```

Comment:
Do we always want to just generate 30 tokens? Feel weird if we are doing a long sequence task like email generation but this can increase speed quite a lot.

Reply:
The inference part in now configurable through the YAML file and accepts the same params as the inference config.